gammapy/background/energy_offset_array.py

```diff
@@ -0,0 +1,137 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+from astropy.units import Quantity, UnitsError
+from astropy.coordinates import Angle
+from astropy.table import Table
+from ..data import DataStore
```

Comment:
I'm also getting this error with your branch:
```
$ python -c 'from gammapy.background import ring_area_factor'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/deil/code/gammapy/gammapy/background/__init__.py", line 5, in <module>
    from .energy_offset_array import EnergyOffsetArray
  File "/Users/deil/code/gammapy/gammapy/background/energy_offset_array.py", line 6, in <module>
    from ..data import DataStore
  File "/Users/deil/code/gammapy/gammapy/data/__init__.py", line 16, in <module>
    from .spectral_cube import *
  File "/Users/deil/code/gammapy/gammapy/data/spectral_cube.py", line 18, in <module>
    from ..spectrum import LogEnergyAxis, powerlaw
  File "/Users/deil/code/gammapy/gammapy/spectrum/__init__.py", line 19, in <module>
    from .spectrum_analysis import *
  File "/Users/deil/code/gammapy/gammapy/spectrum/spectrum_analysis.py", line 12, in <module>
    from ..background import ring_area_factor, Cube
ImportError: cannot import name 'ring_area_factor'
```
The cause is a circular import error within Gammapy (see #171).
One possible fix is to remove the `DataStore` import here.
We said that the `EnergyOffsetArray` class should not use it, so that should be OK, no?